paddle/fluid/inference/tensorrt/engine.h

```diff
@@ -98,6 +100,10 @@ class TensorRTEngine : public EngineBase {
   // LOW EFFICENCY! Get output to CPU, this will trigger a memory copy from GPU
   // to CPU.
   void GetOutputInCPU(const std::string& name, void* dst, size_t max_size);
+  // Fill an ITensor into map itensor_map_.
+  void SetITensor(const std::string& name, nvinfer1::ITensor* tensor);
+  // Get an ITensor called name.
+  nvinfer1::ITensor* GetITensor(const std::string& name);
```

Comment:
Does this interface necessary?

Reply:
If we don't have `GetITensor` interface, we should directly call `auto tensor = itensor_map_[name]` to get the itensor, which has two disadvantages:
- Each time we should call `PADDLE_ENFORCE(itensor_map_.count(name), "no itensor %s", name);` before.
- `itensor_map_` should be public.